gandi/cli/commands/disk.py

```diff
@@ -119,13 +119,18 @@ def detach(gandi, resource, background, force):
 @cli.command()
 @click.option('--bg', '--background', default=False, is_flag=True,
               help='Run command in background mode (default=False).')
+@click.option('-r', '--read-only', default=False, is_flag=True,
+        help='Attach disk as read-only')
+@click.option('--position', '-p',
```

Comment:
you should specify a type INT for this option, this wayy you don't have to cast to int in the module afterwards